public/votes.cfg

```diff
@@ -1,44 +1,36 @@
 # public/votes.cfg
+# Get the maps from https://twelo.pages.dev/maps
```

Comment:
this seems more a comment that should go into the github comments not in the actual file